cortex/client.py

```diff
@@ -199,10 +124,11 @@ class Cortex(object):
     """
 
     @staticmethod
-    def client(api_endpoint: str = None, api_version: int = _DEFAULT_API_VERSION, verify_ssl_cert=None,
+    def client(api_endpoint: str = None, api_version: int = Constants.default_api_version, verify_ssl_cert=None,
                token: str = None, config: dict = None, project: str = None):
         """
-        Gets a client with the provided parameters. All parameters are optional and default to environment variable values if not specified.
+        Gets a client with the provided parameters. All parameters are optional and default to environment variable
+        values if not specified.
```

Comment:
Where do we specify the environment variables in the help section for each param?

Reply:
Didn't understand the question @pkandarpa-cs

Reply:
The text says all parameters are optional and default to environment variables but where are these environment variables specified? 

Like the specific variables that need to be set (like CORTEX_PROJECT) or others

Can we specify them in the parameter descriptions ? Are they specified in an outer readme file

Reply:
I guess we will have to add it in docs and readme 
@smichalski-cs FYI